rskj-core/src/test/java/co/rsk/peg/whitelist/constants/WhitelistConstantsTest.java

```diff
@@ -0,0 +1,64 @@
+package co.rsk.peg.whitelist.constants;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.samePropertyValuesAs;
+
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class WhitelistConstantsTest {
+
+    @ParameterizedTest
+    @MethodSource("getLockWhitelistChangeAuthorizerProvider")
+    void getLockWhitelistChangeAuthorizer(WhitelistConstants whitelistConstants, AddressBasedAuthorizer expectedLockWhitelistChangeAuthorizer) {
+        AddressBasedAuthorizer actualLockWhitelistChangeAuthorizer = whitelistConstants.getLockWhitelistChangeAuthorizer();
+
+        assertThat(actualLockWhitelistChangeAuthorizer, samePropertyValuesAs(expectedLockWhitelistChangeAuthorizer));
+    }
+
+    private static Stream<Arguments> getLockWhitelistChangeAuthorizerProvider() {
+        //MainNet
+        List<ECKey> mainNetLockWhitelistAuthorizedKeys = Arrays.stream(new String[]{
+            "041a2449e9d63409c5a8ea3a21c4109b1a6634ee88fd57176d45ea46a59713d5e0b688313cf252128a3e49a0b2effb4b413e5a2525a6fa5894d059f815c9d9efa6"
+        }).map(hex -> ECKey.fromPublicOnly(Hex.decode(hex))).collect(Collectors.toList());
```

Comment:
Or even
```suggestion
       ECKey authorizerPublicKey = ECKey.fromPublicOnly(Hex.decode("041..."))
       List<ECKey> mainNetLockWhitelistAuthorizedKeys = Collection.singletonList(authorizerPublicKey);
```
It's just 1 key, no need to iterate

Reply:
Same in the other networks

Reply:
Hey, guys! @apancorb @marcos-iov 
Why should we put different values than the production code? With this test, I'm just making sure there are no changes that can affect the production code. You can see WhitelistMainNetConstants, WhitelistRegTestConstants, and WhitelistTestNetConstants, I'm using the same values. Below is a WhitelistMainNetConstants example:

<img width="1206" alt="image" src="https://github.com/rsksmart/rskj/assets/160710714/694f12d7-d47a-4d2c-81aa-da002d467638">

Reply:
@apancorb I know that is an improvement, I agree. If we change the prod code, I think this makes sense, otherwise, we are improving the  UT's code only. The idea of the test is to make sure that the new changes are not breaking the functionality and make more realistic the test, in this case with data close to production. 

@marcos-iov Should we change the production code?